compiler/rustc_passes/src/stability.rs

```diff
@@ -1088,21 +1068,20 @@ fn unnecessary_partially_stable_feature_lint(
              by the feature `{implies}`"
         ),
         |lint| {
-            lint
-        .span_suggestion(
-            span,
-            &format!(
+            lint.span_suggestion(
+                span,
+                &format!(
                 "if you are using features which are still unstable, change to using `{implies}`"
             ),
-            implies,
-            Applicability::MaybeIncorrect,
-        )
-        .span_suggestion(
-            tcx.sess.source_map().span_extend_to_line(span),
-            "if you are using features which are now stable, remove this line",
-            "",
-            Applicability::MaybeIncorrect,
-        )
+                implies,
+                Applicability::MaybeIncorrect,
+            )
+            .span_suggestion(
+                tcx.sess.source_map().span_extend_to_line(span),
+                "if you are using features which are now stable, remove this line",
+                "",
+                Applicability::MaybeIncorrect,
+            )
         },
     );
```

Comment:
I think this one should be derivable, right?

Reply:
Hmm...I never touched this part down here (lines 1090-1107). This was just `cargo fmt` doing it's thing.

However, this certainly looks like it should be migrated to something translateable. Does the linting need to be improved?

I'll add this to my list.

Reply:
If the linter didn't catch this then that would be a bug.

Reply:
The linter does not trigger on this, no.